stdlib/sys/__init__.pyi

```diff
@@ -129,15 +178,35 @@ class _flags(_UninstantiableStructseq, _FlagTuple):
     def utf8_mode(self) -> int: ...
     if sys.version_info >= (3, 10):
         @property
-        def warn_default_encoding(self) -> int: ...  # undocumented
+        def warn_default_encoding(self) -> int: ...
     if sys.version_info >= (3, 11):
         @property
         def safe_path(self) -> bool: ...
+    if sys.version_info >= (3, 10):
+        @property
+        def int_max_str_digits(self) -> int: ...
```

Comment:
This was backported to all our supported versions of Python in patch releases as a security fix:

```pycon
Python 3.8.18 (default, Feb 15 2024, 19:36:58) 
[Clang 15.0.0 (clang-1500.1.0.2.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> sys.flags.int_max_str_digits
-1
```

so I think we should do this, to avoid possible false positives:

```suggestion
    # Whether or not this exists on lower versions of Python
    # may depend on which patch release you're using
    # (it was backported to all Python versions on 3.8+ as a security fix)
    @property
    def int_max_str_digits(self) -> int: ...
```

Reply:
Sure. I think I have the last binary release of each installed locally, so I didn't have it there. That also implies that _FlagTuple for lower versions should also be updated.